src/main/resources/config/tr.yml

```diff
@@ -0,0 +1,359 @@
+# DiscordConsoleChannelFilters: Discord'a gönderilen konsol hatlarına regex filtreleri uygulanacak, sonuç boşsa mesaj hiç gönderilmeyecek
+# DiscordConsoleChannelLevels: Ekleyici aracılığıyla konsol kanalına gönderilecek seviyeler
+# DiscordConsoleChannelUseCodeBlocks: Konsolun kod bloklarına sarılması ve renklendirilmesi gerekiyorsa
+# DiscordConsoleChannelBlockBots: Botların konsol kanalında komut göndermesine izin verilip verilmemesi gerektiği
+
+#
+DiscordConsoleChannelLogRefreshRateInSeconds: 5
+DiscordConsoleChannelUsageLog: "Console-%date%.log"
+DiscordConsoleChannelBlacklistActsAsWhitelist: false
+DiscordConsoleChannelBlacklistedCommands: ["?", "op", "deop", "execute"]
+DiscordConsoleChannelFilters: {".*(?i)async chat thread.*": "", ".*There are \\d+ (?:of a max of|out of maximum) \\d+ players online.*": ""}
+DiscordConsoleChannelLevels: [info, warn, error]
+DiscordConsoleChannelUseCodeBlocks: true
+DiscordConsoleChannelBlockBots: true
+
+# Sohbet kanalı komutu yürütme komutu
+# Bu seçenekler, bir komutu çalıştırmak için "!c kick Notch" veya önek ne olursa olsun deme yeteneğini kontrol eder.
+# Konsol olarak kayıtlı bir sohbet kanalından.
+#
+# DiscordChatChannelConsoleCommandEnabled: Bir sohbet kanalından gelen konsol komutlarına izin verilip verilmeyeceği.
+# DiscordChatChannelConsoleCommandNotifyErrors: İzni olmayan bir komutu izinsiz çalıştırmaya çalışan bir kullanıcıya gönderilip gönderilmeyeceği
+# DiscordChatChannelConsoleCommandPrefix: Konsol komutları için kullanılacak önek. Örneğin. "!c tps"
+# DiscordChatChannelConsoleCommandRolesAllowed: Sohbet kanalından sunucu komutlarını yürütmesine izin verilen kullanıcı rolleri
+# DiscordChatChannelConsoleCommandWhitelist: Çalıştırılabilecek komutların listesi DiscordChatChannelConsoleCommandPrefix
+# DiscordChatChannelConsoleCommandWhitelistBypassRoles: Beyaz listeyi atlayan rollerin listesi
+# DiscordChatChannelConsoleCommandWhitelistActsAsBlacklist: Beyaz liste komutu bunun yerine kara liste görevi görmeli mi?
+# DiscordChatChannelConsoleCommandExpiration: Gönderilen bir komut çıktısının bot tarafından otomatik olarak kaldırılmasına kadar geçen saniye cinsinden süre. Süre sonunu devre dışı bırakmak için 0'a ayarlayın.
+# DiscordChatChannelConsoleCommandExpirationDeleteRequest: Komutu ilk veren kişinin mesajının silinip silinmeyeceği
+#
+DiscordChatChannelConsoleCommandEnabled: true
+DiscordChatChannelConsoleCommandNotifyErrors: true
+DiscordChatChannelConsoleCommandPrefix: "!c"
+DiscordChatChannelConsoleCommandRolesAllowed: ["Owner", "Developer"]
+DiscordChatChannelConsoleCommandWhitelist: ["say", "lag", "tps"]
+DiscordChatChannelConsoleCommandWhitelistBypassRoles: ["Owner", "Developer"]
+DiscordChatChannelConsoleCommandWhitelistActsAsBlacklist: false
+DiscordChatChannelConsoleCommandExpiration: 0
+DiscordChatChannelConsoleCommandExpirationDeleteRequest: true
+
+# Sohbet kanalı oynatıcı listesi komutu
```

Comment:
Auto translate translated "player" incorrectly, use "oyuncu" instead of "oynatıcı"